examples/errortest/errortest.go

```diff
@@ -0,0 +1,110 @@
+package main
```

Comment:
How does `errortest.go` differ from `traversaltime.go`? I see it stores file handles on success, but nothing about this example being related to testing errors. Am I missing something? It feels like the existing example (`traversaltime.go`) with your modifications on reporting errors is all what is needed.

Reply:
To be honest I cant quite recall why I did this so Ill just move it into traversaltime.go

Reply:
Can't quite recall why I split it up.  Ive merged it back and put the file back to where it was before.